pkgs/top-level/all-packages.nix

```diff
@@ -7259,6 +7259,46 @@ with pkgs;
 
   cpcfs = callPackage ../tools/filesystems/cpcfs { };
 
+  advcpmv = let
```

Comment:
To be honest I can't think on a better way to write it.
It is not a `callPackage`-able expression, but a mere overriding of an existing one. 
So I can't imagine how to make it `callPackage`-able.

I could write this in a file and `import` it. But why? It would equally not be callPackage-able.